src/types/date.rs

```diff
@@ -0,0 +1,119 @@
+use std;
+use std::fmt;
+use std::fmt::Debug;
+use std::error::Error;
+use neon_runtime;
+use neon_runtime::raw;
+use context::{Context};
+use context::internal::Env;
+use result::{JsResult, JsResultExt};
+use object::{Object};
+use handle::{Handle, Managed};
+use super::{Value, ValueInternal};
+
+/// A JavaScript Date object.
+#[repr(C)]
+#[derive(Clone, Copy, Debug)]
+pub struct JsDate(raw::Local);
+
+impl Value for JsDate { }
+
+impl Managed for JsDate {
+    fn to_raw(self) -> raw::Local { self.0 }
+
+    fn from_raw(_: Env, h: raw::Local) -> Self { JsDate(h) }
+}
+
+#[derive(PartialEq, PartialOrd, Clone, Debug)]
```

Comment:
It seems strange to implement anything besides `Debug` on this type and possibly `Clone`.

Reply:
I think `Value` requires that JsDate implement `Clone` and `Copy`.

Reply:
https://github.com/neon-bindings/neon/blob/main/src/types/mod.rs#L72-L73